src/clojure/nrepl/middleware/print.clj

```diff
@@ -0,0 +1,176 @@
+(ns nrepl.middleware.print
```

Comment:
It’s generally a good idea to add some high level overview here as well. (And maybe a link to the manual)

Reply:
I've added a one-line summary and referenced the documentation, but I'm not sure if you mean a link to the `adoc` file somehow or just to https://nrepl.org/nrepl/design/middleware.html#_pretty_printing.